tests/test_flows/test_sklearn.py

```diff
@@ -652,110 +678,148 @@ def test_illegal_parameter_names(self):
+             sklearn.preprocessing.OneHotEncoder(sparse=False,
+                                                 handle_unknown='ignore')),
+            ('steps', sklearn.ensemble.BaggingClassifier(
+                base_estimator=sklearn.tree.DecisionTreeClassifier))
         ]
         self.assertRaises(ValueError, sklearn.pipeline.Pipeline, steps=steps)
 
 
     def test_illegal_parameter_names_featureunion(self):
         # illegal name: transformer_list
         transformer_list = [
-            ('transformer_list', sklearn.preprocessing.Imputer(strategy='median')),
-            ('OneHotEncoder', sklearn.preprocessing.OneHotEncoder(sparse=False, handle_unknown='ignore'))
+            ('transformer_list',
+             Imputer(strategy='median')),
+            ('OneHotEncoder',
+             sklearn.preprocessing.OneHotEncoder(sparse=False,
+                                                 handle_unknown='ignore'))
         ]
-        self.assertRaises(ValueError, sklearn.pipeline.FeatureUnion, transformer_list=transformer_list)
+        self.assertRaises(ValueError, sklearn.pipeline.FeatureUnion,
+                          transformer_list=transformer_list)
 
     def test_paralizable_check(self):
-        # using this model should pass the test (if param distribution is legal)
+        # using this model should pass the test (if param distribution is
+        # legal)
         singlecore_bagging = sklearn.ensemble.BaggingClassifier()
         # using this model should return false (if param distribution is legal)
         multicore_bagging = sklearn.ensemble.BaggingClassifier(n_jobs=5)
         # using this param distribution should raise an exception
-        illegal_param_dist = {"base__n_jobs": [-1, 0, 1] }
+        illegal_param_dist = {"base__n_jobs": [-1, 0, 1]}
         # using this param distribution should not raise an exception
         legal_param_dist = {"base__max_depth": [2, 3, 4]}
 
         legal_models = [
             sklearn.ensemble.RandomForestClassifier(),
             sklearn.ensemble.RandomForestClassifier(n_jobs=5),
             sklearn.ensemble.RandomForestClassifier(n_jobs=-1),
-            sklearn.pipeline.Pipeline(steps=[('bag', sklearn.ensemble.BaggingClassifier(n_jobs=1))]),
-            sklearn.pipeline.Pipeline(steps=[('bag', sklearn.ensemble.BaggingClassifier(n_jobs=5))]),
-            sklearn.pipeline.Pipeline(steps=[('bag', sklearn.ensemble.BaggingClassifier(n_jobs=-1))]),
-            sklearn.model_selection.GridSearchCV(singlecore_bagging, legal_param_dist),
-            sklearn.model_selection.GridSearchCV(multicore_bagging, legal_param_dist)
+            sklearn.pipeline.Pipeline(
+                steps=[('bag', sklearn.ensemble.BaggingClassifier(n_jobs=1))]),
+            sklearn.pipeline.Pipeline(
+                steps=[('bag', sklearn.ensemble.BaggingClassifier(n_jobs=5))]),
+            sklearn.pipeline.Pipeline(
+                steps=[('bag', sklearn.ensemble.BaggingClassifier(n_jobs=-1))]),
```

Comment:
Would it make sense to check for `n_jobs=None`, too?

Reply:
This is the first one by default

Reply:
and adding `n_jobs=None` will not make the test pass in scikit-learn < 0.20

Reply:
Okay, got it. Thanks for clarifying this.